homeassistant/components/remember_the_milk/__init__.py

```diff
@@ -137,10 +137,12 @@ def register_account_callback(_):
     request_id = configurator.async_request_config(
         f"{DOMAIN} - {account_name}",
         callback=register_account_callback,
-        description="You need to log in to Remember The Milk to"
-        + "connect your account. \n\n"
-        + 'Step 1: Click on the link "Remember The Milk login"\n\n'
-        + 'Step 2: Click on "login completed"',
+        description=(
+            f"You need to log in to Remember The Milk to"
+            f"connect your account. \n\n"
+            f'Step 1: Click on the link "Remember The Milk login"\n\n'
+            f'Step 2: Click on "login completed"'
```

Comment:
None of these strings have a variable in them. So it should be a normal string, not a fstring.